### ISSUE TITLE
Active tab color inconsistency in Dark Theme

### ISSUE THREAD

**Issue by eakwarren** [musescore/MuseScore#23377]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

Tab panels in the Light theme consistently show active tabs as a lighter color. Below, the Instruments and Analysis tabs are the active tabs.
<img width="590" alt="Screenshot 2024-06-26 at 5 18 51 PM" src="https://github.com/musescore/MuseScore/assets/1768096/7c6dcedf-59bb-44b2-99f2-9f7f18daff3c">

In the Dark theme, however, the tab colors are inconsistent. Same tabs are selected as above, but it appears the * The Lord Of the Rings - tab is active and not the Analysis tab.
<img width="590" alt="Screenshot 2024-06-26 at 5 19 20 PM" src="https://github.com/musescore/MuseScore/assets/1768096/4ed5371c-56a1-414a-98c0-3017f3287d40">

Bonus: The active file/part tab text should be bold to be consistent with the sidebar active tab.

### Steps to reproduce

Enable Dark theme.
Open file and at least one Part

### MuseScore Version

OS: macOS 14.5, Arch.: x86_64, MuseScore Studio version (64-bit): 4.3.2-241630832, revision: github-musescore-musescore-22b46f2

### Regression

I don't know

### Operating system

macOS 14.5,

**Comment by eakwarren** [comment]
@jessjwilliamson, friendly ping. :)

**Comment by eakwarren** [comment]
I implemented this in my dev branch. Here's the before/after with changes to the score tab to unify the interface convention. (All panels and score/parts consistently show active tabs with a darker background and bold text.)

<img width="2954" height="603" alt="Image" src="https://github.com/user-attachments/assets/f61b825c-f0d7-4458-848f-0dc19c808098" />

Here's variations with the score/part tabs the same height as the other panels. With and without the SeparatorLine under the tabs, respectively.

<img width="1882" height="159" alt="Image" src="https://github.com/user-attachments/assets/e4e7db76-701d-48d0-ac86-79235ca97d9b" />

<img width="1469" height="159" alt="Image" src="https://github.com/user-attachments/assets/67cefb8c-50d2-499c-96ee-7780469bb828" />

@avvvvve @jessjwilliamson @bkunda Thoughts?

**Comment by avvvvve** [comment]
@eakwarren From the opinions we polled on this internally, we're split on whether we like the change or not. Yes it's consistent with other tab groups, but there's a case for keeping score & part tabs visually distinct from the others since they are fundamentally a different type of tab than all the others in the app. For instance, the others can be undocked and reordered, while these cannot. 

At some point we intend to allow [multiple scores](https://github.com/musescore/MuseScore/issues/12647) to be open in different tabs as well, which will require rethinking these styles from scratch. I'm hesitant to change them now if we will just change them again soon.

That said, it would help to make a final call if this was a PR that we could test ourselves. Using it feels different than looking at some screenshots. So if you could do that that'd be great!

**Comment by eakwarren** [comment]
@avvvvve @jessjwilliamson See PR above. 

For those following along, specifically 3 changes made:

1. heights are now consistent
2. background color states are now consistent
3. active score/part tab text is now bold, matching panel active state.

<img width="1278" height="640" alt="Image" src="https://github.com/user-attachments/assets/1937d8a8-de30-4797-a0e7-154ff55b7044" />

<img width="1278" height="640" alt="Image" src="https://github.com/user-attachments/assets/3c90f596-d6e7-43d8-a8af-b1d745ff5157" />

**Comment by avvvvve** [comment]
Thanks @eakwarren. Nice to be able to try it out. Could you please revert the height of the tabs to what it was previously?

**Comment by eakwarren** [comment]
@avvvvve see new PR#30794 above.

**Comment by avvvvve** [comment]
@eakwarren Thanks for the tweaks. After some deliberation, we've decided the color should not be changed, at least in this way. The color is intentionally lighter to differentiate the active tab from its surroundings. When made darker, it blends in too much with the note input bar bordering above and the canvas bordering below. Also, the contrast ratio between the inactive/active tabs is stronger as-is, which makes it easier to determine which tab is active at a glance. See the simplified UI to the right:

<img width="840" height="496" alt="Image" src="https://github.com/user-attachments/assets/9f14abbd-686c-4680-8c56-b2d92adaefe1" />

Otherwise, using **bold text** for the active tab is nice, but because the width of the text changes when made bold, the tab widths jump around too when selecting different ones (see video below). We previously solved this issue in the Home/Score/Publish tabs in [this PR](https://github.com/musescore/MuseScore/pull/22132) by setting the width of each inactive tab to the width it _will be_ when selected. Do you think you could take a crack at incorporating this method into these tabs (and revert the colors)?

https://github.com/user-attachments/assets/d0b1b824-9a6d-40c0-a3d5-0288430e4381

**Comment by eakwarren** [comment]
@avvvvve Thanks for the additional info. I'm a little confused though since the colors in your NEW example don't correlate with the colors in the PR#30794 artifact build. (See video.) At any rate, I used the same tab colors as the active/inactive palettes for consistency (which also have a contrast ratio of 1.14), but it sounds like other factors including perception next to proximal elements may be more important. Which I can understand ...however, it seems odd that it's that concerning since the light theme suffers from the same tab to note input toolbar lack of contrast. 

https://github.com/user-attachments/assets/363ca654-b9b7-451d-98a3-111f500e1afd

Perhaps making the active score/part a little darker (#28282a in the top example in the video) would match the current contrast ratio. The tab width issue seems easy enough to handle, but honestly, reverting the colors just doesn't address the original issue as I see it.

> At some point we intend to allow https://github.com/musescore/MuseScore/issues/12647 to be open in different tabs as well, which will require rethinking these styles from scratch. I'm hesitant to change them now if we will just change them again soon.

Sounds like it may be best to wait then. :)

**Comment by avvvvve** [comment]
Yeah, let's just table any color changes until we revisit this for multi-score tabs. Thanks for the time you put into this one, but we're going to close this PR and the associated issue as "by design".

If you'd like to implement the boldness in a separate PR, by all means go for it!
